tic-tac-toe/src/app/index.js

```diff
@@ -0,0 +1,45 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { ConnectedRouter } from 'connected-react-router';
+import { Route, Switch } from 'react-router';
+import { connect } from 'react-redux';
+import Game from '@screens/Game';
+import Login from '@screens/Login';
+import Profile from '@screens/Profile';
+import Topbar from '@components/Topbar';
+import { sessionValidation } from '@config/session';
+import { LOGGEDIN } from '@constants/const';
+
+class App extends Component {
+  constructor(props) {
+    super(props);
+    sessionValidation(props.dispatch);
```

Comment:
as i marked in another PR.. we should not be using constructor. We will talk about it tomorrow